docs/openapi.yaml

```diff
@@ -121,13 +122,18 @@ paths:
       description: 'Create a new access request.
 
 
+        Use the "revoke" query parameter to create a request to revoke access
+
+        instead of a request to grant access.
+
+
         If no "status" is specified in the request body, will use the configured
 
         DEFAULT_INITIAL_STATUS. Because users can only request access to a
 
-        resource once, (username, resource_path) must be unique unless past
+        resource once, each ("username", "resource_path") combination must be
```

Comment:
At a high level, is still user requesting access to a `resource_path` or is it requesting access to a `policy`

Reply:
users are requesting access to policies now